website/server/controllers/api-v3/challenges.js

```diff
@@ -21,18 +21,153 @@ import csvStringify from '../../libs/csvStringify';
 let api = {};
 
 /**
+ * @apiDefine ChallengeLeader Challenge Leader
+ * The leader of the challenge can use this route.
+ */
+
+/**
  * @apiDefine ChallengeNotFound
  * @apiError (404) {NotFound} ChallengeNotFound The specified challenge could not be found.
  */
 
 /**
+ * @apiDefine SuccessfulChallengeRequest
+ * @apiSuccess {UUID} challenge.group._id The group id.
+ * @apiSuccess {String} challenge.group.type Group type: `Guild` or `Party`.
+ * @apiSuccess {String} challenge.group.privacy Group privacy: `Public` or `Private`.
+ * @apiSuccess {String} challenge.name Full name of challenge.
+ * @apiSuccess {String} challenge.shortName Short name of challenge.
```

Comment:
Can you describe this as "A shortened name for the challenge, to be used as a tag" (your wording below, which I like).